Formula/gost.rb

```diff
@@ -0,0 +1,18 @@
+class Gost < Formula
+  desc "GO Simple Tunnel - a simple tunnel written in golang"
+  homepage "https://github.com/ginuerzh/gost"
+  url "https://github.com/ginuerzh/gost/archive/v2.11.1.tar.gz"
+  sha256 "d94b570a7a84094376b8c299d740528f51b540d9162f1db562247a15a89340bf"
+  license "MIT"
+  head "https://github.com/ginuerzh/gost.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/gost"
+  end
+
+  test do
+    system bin/"gost", "-V"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
OK, I will write a test later.